packages/orbit-components/src/ErrorFormTooltip/Tooltip/index.tsx

```diff
@@ -213,32 +89,68 @@ const ErrorFormTooltip = ({
     };
   }, [onShown, isHelp, helpClosable]);
 
+  const isVertical =
+    (attrs.popper && attrs.popper["data-popper-placement"] === "top-start") ||
+    (attrs.popper && attrs.popper["data-popper-placement"] === "top-end")
+      ? "bottom"
+      : "top";
+
+  const cssVars = React.useMemo(
+    () => ({
+      "--error-form-tooltip-position": popper.position,
+      "--error-form-tooltip-top": popper.top,
+      "--error-form-tooltip-left": popper.left,
+      "--error-form-tooltip-right": popper.right,
+      "--error-form-tooltip-bottom": popper.bottom,
+      "--error-form-tooltip-transform": popper.transform,
+    }),
+    [popper],
+  );
+
   return (
-    <StyledFormFeedbackTooltip
+    <div
       id={id}
       ref={tooltipRef}
-      shown={shown}
-      isHelp={isHelp}
-      data-test={dataTest}
       aria-live="polite"
-      inlineLabel={inlineLabel}
-      placement={attrs.popper && attrs.popper["data-popper-placement"]}
-      position={popper.position}
-      top={popper.top}
-      left={popper.left}
-      right={popper.right}
-      bottom={popper.bottom}
-      transform={popper.transform}
+      data-test={dataTest}
+      className={cx(
+        "flex justify-between overflow-visible",
+        "rounded-large py-xs px-sm z-10 box-border",
+        "max-h-none w-[min(calc(100%-20px),_100vw)]",
+        isHelp ? "pe-sm bg-blue-normal" : "bg-red-normal",
+        shown ? "visible opacity-100" : "invisible opacity-0",
+        "duration-fast transition-[opacity,visibility] ease-in-out",
+        "bottom-[var(--error-form-tooltip-bottom)] left-[var(--error-form-tooltip-left)] right-[var(--error-form-tooltip-right)] top-[var(--error-form-tooltip-top)] [position:var(--error-form-tooltip-position)] [transform:var(--error-form-tooltip-transform)]",
+        "lm:w-auto tb:rounded-normal",
+        "[&>img]:max-w-full",
+      )}
+      style={cssVars as React.CSSProperties}
     >
-      <StyledArrow
-        isHelp={isHelp}
+      <div
+        className={cx(
+          "absolute start-[8px]",
+          isVertical ? "bottom-[2px]" : "top-[2px]",
+          inlineLabel && "rtl:start-0",
+          isHelp ? "before:bg-blue-normal" : "before:bg-red-normal",
+          "before:absolute before:h-[8px] before:w-[8px] before:-translate-x-1/2 before:-translate-y-1/2 before:rotate-45",
+        )}
         ref={setArrowRef}
-        inlineLabel={inlineLabel}
-        placement={attrs.popper && attrs.popper["data-popper-placement"]}
       />
-      <StyledTooltipContent ref={contentRef}>{children}</StyledTooltipContent>
+      <div
+        className={cx(
+          "font-base text-normal leading-small text-white-normal flex h-full items-center font-normal",
+          "[&>.orbit-text]:text-white-normal [&>.orbit-text]:text-normal hover:[&>.orbit-text]:text-white-normal focus:[&>.orbit-text]:text-white-normal [&>.orbit-text]:font-normal",
```

Comment:
We have to be mindful about those [&>.class] selectors. For instance, here it won't work for all text-links, if the user decides to pass more than one text-link, it will apply only to the first one

Reply:
Should be "[&_a]:text-white-normal"

Reply:
Right, my bad, thanks